assemblies/fine-tuning-a-model-by-using-kubeflow-training.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="fine-tuning-a-model-by-using-kubeflow-training_{context}"]
+= Fine-tuning a model by using Kubeflow Training
+
+[role='_abstract']
+_Supervised fine-tuning_ (SFT) is the process of customizing a Large Language Model (LLM) for a specific task by using labelled data.
+In {productname-long}, you can use the Kubeflow Training Operator (KFTO) and Kubeflow Training Operator Python Software Development Kit (KFTO SDK) to supervise fine-tune an LLM, by using the HuggingFace SFT Trainer.
```

Comment:
Nitpick - using HuggingFace SFT Trainer is just one option to run finetuning, there are also other tools which user can use to run finetuning using PyTorchJob.